tfhe/docs/getting_started/quick_start.md

```diff
@@ -59,5 +59,6 @@ Refer to the [installation documentation](installation.md) for configuration opt
 The `prelude` pattern provides a convenient way to globally import all important **TFHE-rs** traits at once. This approach saves time and avoids confusion.
 
 ```rust
+# #[allow(unused_imports)]
```

Comment:
does this show up in the gitbook doc ?

unfortunately in the .md files we should not have allow attributes otherwise people reading the doc will see those appear in the HTML most likely

Reply:
Apparently not
I tried a few different ways but it didn't work

Reply:
can it be added in modules except docs ? like in high_level, integer, shortint, core ? or is it a always a lib attribute ?

Reply:
it's an attribute so unless we do it on a per module basis I don't think we can undo it, what a lame way to enable stuff

Reply:
I can try

Reply:
Error says "error: this attribute can only be applied at the crate level"

Reply:
🙄 what a weird design